migrations/database_migration.py

```diff
@@ -0,0 +1,136 @@
+import json
+import sqlite3
+from sqlalchemy import create_engine, text
+from sqlalchemy.orm import sessionmaker
+from database.database_tables import Base, User, Role, Balance, Operation, UserWeekStat
+from config.database_config import DATABASE_FILE_PATH, DB_HOST, DB_PORT, DB_USER, DB_PASS, DB_NAME
+from migrations.def_calculate_profit import calculate_week_user_profits_percent
+
+
+def transfer_data():
+    # SQLite
+    sqlite_db_path = DATABASE_FILE_PATH
```

Comment:
This must be an argument of this script, because SQLite DB relates only to this script. Please see https://docs.python.org/3/library/argparse.html